### DISCUSSION TITLE
Does anyone know how to disable IMT Lazarus?

### DISCUSSION THREAD

**Discussion by Duck-relaxing** [3kh0/ext-remover#571] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
IMTLazarus is used at my school, does anyone know how to disable it?
here is the url of the extension:
[https://chrome.google.com/webstore/detail/imtlazarusv16/cgigopjakkeclhggchgnhmpmhghcbnaf](https://chrome.google.com/webstore/detail/imtlazarusv16/cgigopjakkeclhggchgnhmpmhghcbnaf)
the point blank does not work because the pages are reloaded every time the extension starts and I have version 108, I already tried the downgradin but it doesn't work either

**Comment by siouxxsies** [comment, marked as answer]
use the LTBEEF method in the description or the point blank method, i dont get why would you even ask this because the solution is right in front of your big forehead lol

**Reply by Duck-relaxing** [reply]
the point blank does not work because the pages are reloaded every time the extension starts and I have version 108, I already tried the downgradin but it doesn't work either

**Reply by siouxxsies** [reply]
> the point blank does not work because the pages are reloaded every time the extension starts and I have version 108, I already tried the downgradin but it doesn't work either

maybe try using a browser emulator on replit or something because the only real solution to remove a forced extension was LTbeef and the point black method

**Reply by Aka-but-nice** [reply]
> the point blank does not work because the pages are reloaded every time the extension starts and I have version 108, I already tried the downgradin but it doesn't work either

Aren't you the one from TN, also, not much you can do, you can continue trying the exploit in TN, which is not public and I'll not be explaning, if you want information, join [The TN Discord](https://discord.gg/unblock)

**Comment by amelia-bartlett** [comment]
you could either use sh1mmer (if its a chromebook, which I assume it is) or you could reverse engineer the extension and look for something exploitable

**Comment by sayborduu** [comment]
Maybe you go to “El Vedat”? If you do DM me on Discord “justsayb#9170”
